package.json

```diff
@@ -36,6 +36,7 @@
     "react-dom": "17.0.2",
     "react-router": "5.3.3",
     "react-router-dom": "5.3.0",
+    "react-router-dom-v5-compat": "^6.22.0",
```

Comment:
So if I understand this right, this library is like an intermediate migration kit.
In that case this will be a temporary solution right?
I mean, after some point, we need to start using this: https://www.npmjs.com/package/react-router-dom

Reply:
Yeah, that's correct. When i import useParams from `react-router-dom`, there are namespace readability issues. However, i noticed in the dynamic-plugin-sdk code, they imported useParams from `react-router-dom-v5-compat` dependency, which resolves the problem. So, i thought of switch to that dependency to fix the issue with our plugin.

However, I noticed that someone has already created this issue with useParams in the ocp-bugs: 
https://issues.redhat.com/browse/OCPBUGS-29526

Reply:
great! thanks for the clarification

Reply:
dynamic-sdk team has started using  lib `react-router-dom-v5-compat`  instead `react-router`. As we are dependent on dynamic-sdk so let's use same library. keep watch what libs dynamic-sdk team is using. We will have to change this lib in case dynamic-sdk team will change it otherwise continue.